LogAndStream/python_scripts/Shimmer_common/shimmer_comms_bluetooth.py

```diff
@@ -168,6 +168,8 @@ class BtCmds:
     SET_SD_SYNC_COMMAND = 0xE0
     SD_SYNC_RESPONSE = 0xE1
     ACK_COMMAND_PROCESSED = 0xFF
+    PRESSURE_CALIBRATION_COEFFICIENTS_RESPONSE = 0xA6
+    GET_PRESSURE_CALIBRATION_COEFFICIENTS_COMMAND = 0xA7
```

Comment:
Better to move this below line no. 167 "0xA5".

Reply:
agreed and done